sentry/src/main/java/io/sentry/SentryClient.java

```
@@ -120,12 +125,18 @@ private boolean shouldApplyScopeData(
                 SentryLevel.DEBUG,
                 "Event was dropped as the exception %s is ignored",
                 event.getThrowable().getClass());
+        options
+            .getClientReportRecorder()
+            .recordLostEvent(DiscardReason.EVENT_PROCESSOR, DataCategory.Error, options);
```

Ignored exceptions are recorded as reason `EVENT_PROCESSOR` as documented here: https://develop.sentry.dev/sdk/client-reports/#envelope-item-payload